src/WebJobs.Extensions.NotificationHubs/Bindings/NotificationHubAttributeBindingProvider.cs

```diff
@@ -59,9 +59,12 @@ public Task<IBinding> TryCreateAsync(BindingProviderContext context)
 
             Func<string, INotificationHubClientService> invokeStringBinder = (invokeString) => service;
 
+            var converterManager = _jobHostConfig.GetService<IConverterManager>();
+            converterManager = converterManager.AddNotificationHubConverters(attribute.Platform);
```

Comment:
converterManager is global to the host. So if you have 5 function definitions with 5 different attributes, each with a different Platform setting, they'll all overwrite each other.
We need to find some more generalized way to do this with the converter.